Java-EE/security-jwt-example/nb-configuration.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
There is no need for this file, and example should work on any IDE without specifying a configuration for it on the project.